src/types.js

```diff
@@ -94,3 +94,7 @@ export type TextareaState = {
   selectionStart: number,
   component: ?React$StatelessFunctionalComponent<*>,
 };
+
+export type OutputItem = {
+  key: string,
```

Comment:
I'm new to flow, this should be a string or number, would `?string` work for that?

Reply:
Yeah! `?string` is fine! 👍